privacyidea/lib/resolvers/LDAPIdResolver.py

```diff
@@ -438,9 +438,9 @@ def _ldap_attributes_to_user_object(self, attributes):
                 if ldap_k == map_v:
                     if ldap_k == "objectGUID":
                         ret[map_k] = ldap_v[0]
-                    elif type(ldap_v) == list and map_k not in ["mobile"]:
-                        # All lists (except) mobile return the first value as
-                        #  a string. Mobile is returned as a list
+                    elif type(ldap_v) == list and map_k not in ["mobile"] and map_k not in ["memberOf"]:
```

Comment:
Please change this to:

    elif type(ldap_v) == list and map_k not in  ["mobile", "memberOf"]:

We could even define a list of Multivalue-Attributes in the resolver. This way we could be more flexible in the future. 

     MULTI_VALUE_ATTRIBUTE = ["mobile", "memberOf"]

     elif type(ldap_v) == list and map_k not in MULTI_VALUE_ATTRIBUTE:

...and we could keep the backward compatiblity.